.github/workflows/go-getter.yml

```diff
@@ -1,6 +1,11 @@
 name: go-getter
 
-on: [push]
+on: 
+  push:
+    branches:
+      - v2
+  pull_request:
```

Comment:
Out of curiosity what does this empty section imply?

Reply:
I presume it means that it runs on all PRs pushing to v2?

Reply:
Ah good call out. This will actually run for all pull requests. I can update for any pull_requests against v2

Reply:
Updated